hotel-reservation-system/src/components/Filters/HotelFilter.js

```diff
@@ -0,0 +1,59 @@
+import { React, useState, useEffect } from "react";
+import Grid from "@material-ui/core/Grid";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import { TextField } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+import API from "./../../api";
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    margin: 15,
+    alignSelf: "center",
+    alignContent: "center",
+    justifyContent: "center",
+  },
+  button: {
+    margin: 10,
+  },
+}));
+
+export default function HotelFilter({ getValuesFromFilter }) {
+  const classes = useStyles();
+  const [hotelNames, setHotelNames] = useState([]);
+  const [hotelName, setHotelName] = useState("");
+
+  useEffect(() => {
+    loadHotelNames();
+    getValuesFromFilter(hotelName);
+  }, [hotelName]);
+
+  const loadHotelNames = async () => {
+    await API.get("/hotels/hotelNames")
+      .then((response) => response.data)
+      .then((data) => {
+        if (!!data) setHotelNames(data);
+      })
+      .catch((error) => console.log(error));
+  };
+
+  return (
+    <Grid item>
+      <Autocomplete
+        id="hotelNames"
+        options={hotelNames}
+        onChange={(event, value) => {
+          setHotelName(value);
+        }}
+        getOptionLabel={(option) => option}
```

Comment:
Can't you just pass the option as a param?

Reply:
deleted